econml/policy/_drlearner.py

```diff
@@ -98,6 +98,24 @@ def predict_value(self, X):
         """
         return self.drlearner_.const_marginal_effect(X)
 
+    def predict_proba(self, X):
+        """ Predict the probability of recommending each treatment
+
+        Parameters
+        ----------
+        X : {array-like} of shape (n_samples, n_features)

```

Comment:
Why the curly brackets?
```suggestion
        X : array-like of shape (n_samples, n_features)

```